app/models/workload.py

```diff
@@ -0,0 +1,75 @@
+from typing import Dict, Any, Union, Tuple
+
+from sqlalchemy import Column, String, Float
+
+from app import wrapper
+
+
+class Workload(wrapper.Base):
+    """
+    This is the workload-model for cryptic-game
+    """
+
+    __tablename__: str = "workload"
+
+    uuid: Union[Column, str] = Column(String(36), primary_key=True, unique=True)
+
+    performance_cpu: Union[float, Column] = Column(Float)
```

Comment:
Add `nullable=False`
To all Column

Reply:
Not too every one ... I left out usage. -> Done